subsys/bluetooth/host/keys.c

```diff
@@ -279,6 +279,22 @@ void bt_keys_add_type(struct bt_keys *keys, int type)
 	keys->keys |= type;
 }
 
+void bt_keys_clear_other_keys_with_identity(uint8_t id, const bt_addr_le_t *id_addr,
+		struct bt_keys *keep_these_keys)
```

Comment:
The alignment of the split line doesn't look quite right. (I wish we had CI catch this stuff, but it seems Zephyr subsystems are not very consistent with enforcing split line alignment style)

Reply:
> The alignment of the split line doesn't look quite right. (I wish we had CI catch this stuff, but it seems Zephyr subsystems are not very consistent with enforcing split line alignment style)

I think that would be part of the clang-format we are discussing about adding in Discord :)

Reply:
Ok. Yeah, it would be nice with a configuration file for one of the formatting tools.